src/main/java/com/example/CourseWork2/controller/JavaController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.CourseWork2.controller;
+
+
+import com.example.CourseWork2.entity.Question;
+import com.example.CourseWork2.service.JavaQuestionService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaController {
+    private final JavaQuestionService javaQuestionService;
+
+    public JavaController(JavaQuestionService javaQuestionService) {
```

Comment:
```suggestion
    public JavaController(QuestionService questionService) {
```
инжектить надо по интерфейсу всегда